src/util/irep_ids.cpp

```diff
@@ -20,7 +20,7 @@ const char *irep_ids_table[]=
 #define IREP_ID_ONE(id) #id,
 #define IREP_ID_TWO(id, str) #str,
 
-#include "irep_ids.def"
+#include "ids/irep_ids.def"
```

Comment:
Why is a PR that's supposedly only about the `codeowners` file changing these?

Reply:
(As the description above states, "In order to remove the code owner from irep_ids.def, this file was moved into its own subfolder of src/utils.")  We want to give an owner to all the other files in src/utils, and we can't override that rule to remove the owner from a single file.  However, it is possible to set an owner for all files in src/utils but not its subfolders.  We moved irep_ids.def into a separate folder so that it can have its own (empty) list of owners.

Reply:
Makes sense. Thanks.